### DISCUSSION TITLE
mame nonsupport Deathsmiles 2

### DISCUSSION THREAD

**Discussion by ruguoxingxingbukuaile** [mamedev/discussions#120] [Q&A] [answered]
When I use android mame open Deathsmiles 2，It shows“this system doesn't work，The emulation for this system is not yet complete.There is nothing you can do to fix this problem except wait for the developers to improve the emulation”.so，Is there a plan to support the development of this game？

**Comment by allkern** [comment, marked as answer]
Deathsmiles II runs on the Cave PC hardware, which is basically a full Athlon-based PC with a Radeon graphics card. x86 processors are insanely bloated and thus require a lot of work to emulate in general, I wouldn't expect this machine to work anytime soon

**Reply by angelosa** [reply]
It's also a PCI Express MB, which is a world of pain on its own.
And, even if emulation advances at all then performance will be a concern for desktop host PCs already ...

**Comment by ruguoxingxingbukuaile** [comment]
thanks a lot.

**Comment by rb6502** [comment]
Yeah, the original hardware there is more powerful than any Android device, so even if it ran on PCs it wouldn't there.
